src/lib.rs

```diff
@@ -95,7 +97,7 @@ impl AmbiguousLanguages {
 	}
 
 	/// Returns a vector of the possible languages.
-	#[cfg(feature = "std")]
+	#[cfg(any(feature = "alloc", feature = "std"))]
```

Comment:
Shouldn't "std" imply "alloc"?

Reply:
Actually `std` and `alloc` do not rely on each other. Any of them can be enabled independently that's why we need `any` here:
https://github.com/rust-bitcoin/rust-bip39/blob/9180d5697a9577b213846bfe8f66d83039be01f0/Cargo.toml#L15-L18

My PR/commit description was bad, I fixed that.

Reply:
I believe his point was that we can enable "alloc" in "std" (as is common).
```rust
[features]
default = [ "std" ]
std = [ "alloc", "serde/std", "unicode-normalization/std" ]
rand = [ "crate_rand", "rand_core" ]
alloc = [ "unicode-normalization" ]
```

Then use the more simple
```rust
#[cfg(feature = "alloc")]
```

Reply:
done